client-app/src/constants/Enums.js

```diff
@@ -0,0 +1,9 @@
+const ItemStatus = Object.freeze ({
+    DONATED: 'Donated',
+    IN_STORAGE: 'In Storage Facility',
+    REFURBISHED: 'Refurbished',
+    SOLD: 'Item Sold',
+    RECEIVED: 'Received'
+});
+
+export default ItemStatus;
```

Comment:
The ItemStatus constant is well-defined using Object.freeze() to ensure immutability, which is a good practice for maintaining consistent item status values across the application. This will  help the user prevent accidental modifications of the status values, ensuring the integrity of the data. Nice way to wrap up this file inside the src folder.